vaadin-grid-table.html

```diff
@@ -75,37 +74,22 @@
       thead th,
       tfoot td {
         top: 0;
-        background-color: #fff;
-        font-weight: 500;
       }
 
       td, th {
-        background-color: #fff;
-        padding: 0 !important;
+        padding: 0;
         flex-shrink: 0;
         flex-grow: 1;
-        flex-basis: 100px;
         box-sizing: border-box;
         display: flex;
-        overflow: hidden;
       }
 
-      /* Selector for non-shadow */
-      td::content > .cell-content,
-      th::content > .cell-content,
-      /* Selector for shadow */
-      ::content > .cell-content {
-         padding: 8px;
+      td ::content > .cell-content,
+      th ::content > .cell-content {
          width: 100%;
-         box-sizing: border-box;
-         display: flex;
-         justify-content: center;
-         flex-direction: column;
+         display: inline-flex;
          white-space: nowrap;
```

Comment:
Well that’s interesting. Would be nice to have the default text truncation for textual content, like we do in the classic grid.